Duplicati/Library/SQLiteHelper/SQLiteLoader.cs

```diff
@@ -99,6 +99,37 @@ public static System.Data.IDbConnection LoadConnection()
             return con ?? throw new InvalidOperationException("Failed to load connection");
         }
 
+        /// <summary>
+        /// Applies user-supplied custom pragmas to the SQLite connection
+        /// </summary>
+        /// <param name="con">The connection to apply the pragmas to.</param>
+        /// <returns>The connection with the pragmas applied.</returns>
+        public static System.Data.IDbConnection ApplyCustomPragmas(System.Data.IDbConnection con)
+        {
+            var opts = Environment.GetEnvironmentVariable("CUSTOMSQLITEOPTIONS_DUPLICATI");
+            if (opts == null)
+                return con;
+            var topts = opts.Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+            if (topts.Length == 0)
+                return con;

```

Comment:
Maybe a warning indicating that whatever they put in the variable didn't parse correctly? I guess that the only case where this is hit is where the variable has been set to 0 or more semicolons, and nothing else.